Hymns/Tags/TagStore.swift

```diff
@@ -7,7 +7,7 @@ protocol TagStore {
     func storeTag(_ entity: TagEntity)
     func deleteTag(primaryKey: String, tag: String)
     func getSongsByTag(_ tag: String) -> [SongResultViewModel]
-    func getTagsForHymn(hymnIdentifier: HymnIdentifier) -> [String]
+    func getTagsForHymn(hymnIdentifier: HymnIdentifier) -> ([String], [TagColor])
```

Comment:
Instead of a Tuple of Arrays, can you just have an array of tuples?

Reply:
Addressed in recent commit